config-example.yml

```diff
@@ -0,0 +1,140 @@
+graphhopper:
+
+  # OpenStreetMap input file
+  datareader.file: core/files/monaco.osm.gz
+
+  ##### Vehicles #####
+
+
+  # Possible options: car,foot,bike,bike2,mtb,racingbike,motorcycle (comma separated)
+  # bike2 takes elevation data into account (like up-hill is slower than down-hill) and requires enabling graph.elevation.provider below
+  graph.flag_encoders: car
+
+
+  # Enable turn restrictions for car or motorcycle.
+  # graph.flag_encoders: car|turn_costs=true
```

Comment:
This is kind of a bridge technology anyway -- at one point, we should migrate it to a hierarchical config without self-invented encoding of structure.

I'll think about a suggestion what to do in the meantime.